Orange/ensembles/ada_boost.py

```diff
@@ -1,24 +1,37 @@
 import sklearn.ensemble as skl_ensemble
-from Orange.base import SklLearner
-from Orange.classification.base_classification import (SklLearnerClassification,
-                                                       SklModelClassification)
-from Orange.regression.base_regression import (SklLearnerRegression,
-                                               SklModelRegression)
 
-__all__ = ["SklAdaBoostLearner", "SklAdaBoostRegressionLearner"]
+from Orange.base import SklLearner, SklModel
+from Orange.classification.base_classification import (
+    SklLearnerClassification, SklModelClassification
+)
+from Orange.modelling import Fitter
+from Orange.regression.base_regression import (
+    SklLearnerRegression, SklModelRegression
+)
+
+__all__ = [
+    'SklAdaBoostLearner',
+    'SklAdaBoosClassificationLearner',
+    'SklAdaBoostRegressionLearner',
+]
 
 
 class SklAdaBoostClassifier(SklModelClassification):
     pass
 
 
-class SklAdaBoostLearner(SklLearnerClassification):
+class SklAdaBoosClassificationLearner(SklLearnerClassification):
     __wraps__ = skl_ensemble.AdaBoostClassifier
     __returns__ = SklAdaBoostClassifier
-    name = 'skl adaBoost'
+    name = 'skl adaBoost classification'
```

Comment:
My bad, I must've missed when this was added. So I should just skip defining the `name` attributes altogether?